Source/WebCore/platform/graphics/cg/ImageBufferCGBitmapBackend.cpp

```diff
@@ -54,28 +53,31 @@ std::unique_ptr<ImageBufferCGBitmapBackend> ImageBufferCGBitmapBackend::create(c
     if (backendSize.isEmpty())
         return nullptr;
 
-    void* data;
     unsigned bytesPerRow = 4 * backendSize.width();
```

Comment:
Why start checking only _after_ multiplying by 4? We can start using `CheckedSize` a little sooner.

Reply:
Indeed, I missed that. I'll fix before landing.